src/fragmentarium/ui/image-annotation/Annotator.tsx

```diff
@@ -153,9 +153,9 @@ const WithAnnotations = withData<
 const WithPhoto = withData<
   { fragment: Fragment; fragmentService: FragmentService },
   unknown,
-  Blob
+  Blob | null
 >(
-  ({ data, ...props }) => <WithAnnotations {...props} image={data} />,
+  ({ data, ...props }) => <WithAnnotations {...props} image={data!} />,
```

Comment:
Avoid `!`. If null is given but not allowed, throw an exception.

Reply:
should the exception throw in the `WithAnnotations` component or in the within the `withData`